src/ol/control/ZoomToExtent.js

```diff
@@ -15,6 +15,7 @@ import {fromExtent as polygonFromExtent} from '../geom/Polygon.js';
  * Instead of text, also an element (e.g. a `span` element) can be used.
  * @property {string} [tipLabel='Fit to extent'] Text label to use for the button tip.
  * @property {import("../extent.js").Extent} [extent] The extent to zoom to. If undefined the validity
+ * @property {Object<string, *>} [properties] Arbitrary observable properties. Can be accessed with `#get()` and `#set()`.
  * extent of the view projection is used.
```

Comment:
This line also belongs to the previous property description

Reply:
Fixed.